pyproject.toml

```diff
@@ -30,8 +28,15 @@ dependencies = [
     'sympy>=1.0',
     'numba>=0.35',
     'dill>=0.2.6',
+    'qmat',
```

Comment:
I guess that should have `>=0.1.8` here, too?

Reply:
Done in the next commit